sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -389,15 +392,48 @@ async function main() {
 main();
 ```
 
+### Dynamic Classification
+
+On the fly classification of the input documents into one or multiple categories. Assigns either one or multiple categories per document. This type of classification doesn't require model training.
```

Comment:
> either one or multipole categories

This is what concerns me. Do I correctly assume the response will always contain an array, though? Not only should REST responses not vary the response property type unless the containing model (or ancestor) is polymorphic, but it would also be a problem for typed languages.

Reply:
Yes, the response will always be an array.